bin/dde4_update_script.sh

```diff
@@ -58,9 +55,6 @@ echo -e '\e[1;33mUpdating new configurations..\e[0m'
 sed -i -e "/^production:/,/database:/{/^\([[:space:]]*database: \).*/s//\1${PRODUCTION_DB}/}" \
     -e "/^production:/,/username:/{/^\([[:space:]]*username: \).*/s//\1${DDE_DB_USERNAME}/}" \
     -e "/^production:/,/password:/{/^\([[:space:]]*password: \).*/s//\1${DDE_DB_PASSWORD}/}" \
-    -e "/^:dde_sync_config:/,/:protocol:/{/^\([[:space:]]*:protocol: \).*/s//\1${SYNC_PROTOCOL}/}" \
-    -e "/^:dde_sync_config:/,/:host:/{/^\([[:space:]]*:host: \).*/s//\1${SYNC_HOST}/}" \
```

Comment:
This section should be required

Reply:
If we are to use SSL syncing then protocol to be used is https. So if we are to be reading current database.yml  which uses http it will not work.  Same with the host for current configurations its always the master IP address, so with the decision made to use SSL we are hitting the dashboard for reverse proxy syncing. 

Please let me know if this is not clear.